.github/workflows/cd.yml

```diff
@@ -38,17 +38,16 @@ jobs:
         run: npm test
 
       # Deploy to GitHub Pages
-      - name: Commit distribution artifacts
+      - name: Prepare deployment
         run: |
-          git checkout --orphan dist
-          git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
-          git config --local user.name "github-actions[bot]"
-          git reset
-          git add -f ./distribution/index.html
-          git commit -m "Deploy site"
-      - name: Push changes
-        uses: ad-m/github-push-action@master
+          rm ./out/tscript.js
```

Comment:
Why does it get removed?

Reply:
This is the build output from `rpm run interop`. It's not used in the IDE and is just produced to test that it builds without errors. It's not uploaded currently and I don't really see a good reason to change that.